src/dump.ts

```diff
@@ -21,7 +21,7 @@ export function dump(
 ): RegexDoctorResult {
   const map = doctor.map
   const {
-    limitCalls = 10,
+    limitCalls = Number.POSITIVE_INFINITY,
```

Comment:
This was not about the rendering perf, but more like the dumping of huge amounts of data would cause the Node.js to exceed the memory limit. Infinity is not a good default to me.

Reply:
If `calls` is limited to 10, rendering perf will lose its meaning. In addition, I have not reproduced the OOM problem. I think it has more to do with the length of the input string.

However, if the dump does not return all data, the subsequent feature of "dynamically modifying the regex and re-running the test" will not be possible.

But, you are right, it is not about the rendering perf. So, I will undo this changed.